core/web/api/analytics.py

```diff
@@ -1,17 +1,24 @@
 from __future__ import unicode_literals
 
+import logging
 from flask_login import current_user
 from flask_classy import route
 from flask import request
 
+from core.user import User
 from core.observables import Observable
 from core.web.api.crud import CrudApi
 from core import analytics
+from core.config.config import yeti_config
 from core.analytics_tasks import schedule
 from core.web.api.api import render
 from core.web.helpers import get_object_or_404
 from core.web.helpers import requires_permissions
 
+shared_keys = False
+if hasattr(yeti_config, "shared_keys"):
+    shared_keys = yeti_config.shared_keys.enabled
```

Comment:
I think you can do `shared_keys = yeti_config.get("shared_keys", "enabled", False)`

Reply:
done